R/util-render_rmd.R

```diff
@@ -0,0 +1,24 @@
+#' Custom Rmarkdown render function
+#'
+#' Rmarkdown render function that defaults to rendering intermediate rmd files in a temporary directory
+#'
+#' @param input path to the template Rmd file
+#' @param output_file path to the location where output will be saved
+#' @param params list of params to pass to the template Rmd file
```

Comment:
```suggestion
#' @param strInputPath `string` or `fs_path` Path to the template `Rmd` file.
#' @param strOutputFile `string` Filename for the output.
#' @param strOutputDir `string` or `fs_path` Path to the directory where the output will be saved.
#' @param lParams `list` Parameters to pass to the template `Rmd` file.
```

We're exporting, so fully document.